specifications/Prisoners_Single_Switch/Prisoner.cfg

```diff
@@ -0,0 +1,12 @@
+CONSTANTS
+  Prisoner = {"Alice", "Bob", "Eve"}
+  Designated_Counter = "Alice"
+  Light_Unknown = FALSE
+
+SPECIFICATION Spec
+PROPERTY Terminating
+
+INVARIANT TypeOK
+INVARIANT VictoryOK
+
+CHECK_DEADLOCK FALSE
```

Comment:
The ITEs in `StandardAction` and `CounterAction` ensures that the actions remain permanently enabled. In other words, this spec cannot deadlock, which is why this check is not relevant.